contracts/factory/src/contract.rs

```diff
@@ -75,6 +79,10 @@ impl FactoryTrait for Factory {
             panic!("Factory: Initialize: there must be at least one whitelisted account able to create liquidity pools.")
         }
 
+        if lp_token_decimals < 6 {
+            panic!("Factory: Initliaze: trying to create factory with invalid param: lp_token_demicals < 6");
+        }
+
```

Comment:
Why is this for? Remove it.

Reply:
I wanted to make sure we don't set some wrongful number for decimals. It's removed